Fixed Names of Pokemon

gdirk07 opened the pull request:
Names in the Pokemon API only used "-" for special characters so I created a function to apply the proper name as the series gave them.

To help with this I created a subdirectory "utils" which contain
1. hardcoded values for special cased pokemon
2. A file we can use to store helper functions.
This structure is subject to change but for the time being it works

Resolves issue #28

gdirk07 commented on pokemon-checker/src/components/PokemonDisplay/PokemonDisplay.tsx:
```diff
@@ -27,6 +27,9 @@ class PokemonDisplay extends React.Component<displayProps, displayState> {
   //TODO (jeremy): Move this factory to a service! Views shouldn't control this.
   private pokemonFactory: PokemonFactory;
 
+  private readonly width: string = "96";
+  private readonly height: string = "96";
+
```
ah crap forgot I worked with this, should be in another branch instead

gdirk07 commented:
I think something to potentially explore is how special characters (♂, ♀ etc) will have an affect on various browser and potentially devices.

gdirk07 commented on pokemon-checker/src/components/PokemonDisplay/PokemonDisplay.tsx:
```diff
@@ -29,6 +29,9 @@ class PokemonDisplay extends React.Component<displayProps, displayState> {
   //TODO (jeremy): Move this factory to a service! Views shouldn't control this.
   private pokemonFactory: PokemonFactory;
 
+  private readonly width: string = "96";
+  private readonly height: string = "96";
```
```suggestion
```

jeremy-jtlo commented:
Just got back to this

If we want to continue this route, can we move any constants to the `constants/` directory? I'd save `utils/` for classes that do things not necessarily bound to any of our data types.

gdirk07 commented:
Missed the comment. I moved the name exception constants to constants/ and renamed Helper into a more specific name because both APP and Factory currently use it. If we ever reference Pokemon in the API for the purpose of displaying in the UI we will need the API to fix the name. I'll leave it up to you if we keep that in utils/ or move it somewhere else
